src/components/chartRepo/ChartRepo.tsx

```diff
@@ -216,10 +216,14 @@ function ChartForm({ id = null, name = "", active = false, url = "", authMode =
 
             if (result && !result?.actualErrMsg) {
                 setValidationStatus(VALIDATION_STATUS.SUCCESS)
-                toast.success('Successfully saved.')
+                toast.success(
+                    <ToastBody
+                        title="Chart repo added."
```

Comment:
added word is misleading as on update also we are calling the same function.
Please connect with utkarsh for generic messaging

Reply:
same with subtitle ("chart to be available" sentence is misleading for update as chart is already available. )

Reply:
Done. Changed title to -> `Chart repo saved` & using `listed` instead of `available` in subtitle.